pkg/streaming/commands/kafkagateway_delete.go

```diff
@@ -72,12 +72,12 @@ func NewKafkaGatewayDeleteCommand(ctx context.Context, c *cli.Config) *cobra.Com
 		Use:   "delete",
 		Short: "delete kafka gateway(s)",
 		Long: strings.TrimSpace(`
-Delete one or more kafka gateways by name or all kafka gateways within a
+Delete one or more Kafka Gateways by name or all Kafka Gateways within a
 namespace.
 
-Deleting a kafka gateway will disrupt all processors consuming streams managed
-by the gateway. Existing messages in the stream may be preserved by the
-underlying kafka broker, depending on the implementation.
+Deleting a Kafka Gateway will disrupt all Processors consuming streams managed
+by the Gateway. Existing messages in the stream may be preserved by the
+underlying Kafka broker, depending on the implementation.
```

Comment:
This is where capitalizing CRDs get confusing, because `Kafka` is not a CRD

Reply:
now fixed everywhere